### ISSUE TITLE
grammar "Processing" and extension "pde"

### ISSUE THREAD

**Issue by bgc** [Glavin001/atom-beautify#468]
Is it possible to add support for grammar "Processing" and extension "pde"

**Comment by bgc** [comment]
If not, could you give some pointers on how to implement this feature?

**Comment by Glavin001** [comment]
@bgc : this would be the documentation for adding the Language and adding a Beautifier that supports it: https://github.com/Glavin001/atom-beautify/blob/master/docs/add-languages-and-beautifiers.md

**Comment by ericoporto** [comment]
isn't a pde file just a JAVA file? Or this assumes the sketch.properties always are filled with mode=Java ?

**Comment by bgc** [comment]
@ericoporto yes. i've tried it with uncrustify (the one atom-beautify uses for JAVA) and it works. just need to test my fork in atom to check if all is ok.

**Comment by kleerkoat** [comment]
@bgc how did this turn out? ty
